components/AmountInput/AmountInput.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import 'styled-components/macro'
+
+import anjColor from './anj-color.svg'
+import anjWhite from './anj-white.svg'
+import antColor from './ant-color.svg'
+import antWhite from './ant-white.svg'
+
+function getImage(color, symbol) {
+  if (symbol === 'ANT') {
+    return color ? antColor : antWhite
+  }
+  if (symbol === 'ANJ') {
+    return color ? anjColor : anjWhite
+  }
+}
+
+function AmountInput({
+  color = true,
+  disabled = false,
+  symbol,
+  value,
+  onChange,
+}) {
+  return (
+    <label
+      css={`
+        font-family: 'Manrope';
```

Comment:
We should at least add `sans-serif` afterwards, so that a proper replacement is used while the font is being downloaded or if it doesn’t load at all.